ledger/persistedboxes_list.go

```diff
@@ -0,0 +1,143 @@
+// Copyright (C) 2019-2022 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package ledger
+
+// persistedBoxesDataList represents a doubly linked list.
+// TODO: must initiate with newPersistedAccountList.
```

Comment:
@algoidurovic Is the TODO still relevant?

I leave a _suggestion_ to update (what appears to be) a stale comment reference.

```suggestion
must initiate with newPersistedBoxList.
```

Reply:
I wasn't sure what the purpose of this this TODO was. The resource cache had the same comment so I copied it over. Maybe @algorandskiy has some input?

Reply:
@algoidurovic Where do you see the TODO in resource cache?  I don't see a corresponding TODO in `persistedresources_list.go`.

Reply:
Sorry, the TODO wasn't in `persistedresources_list.go`, the comment itself was. The TODO was added by me to look into if this comment would also apply to `newPersistedBoxList`. On second inspection, it looks like the original comment in the resources file was erroneously copied over from the account file. I'll change that as well.